tests/bpf2c_plugin/bpf2c_test.cpp

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define WIN32_LEAN_AND_MEAN // Exclude rarely-used stuff from Windows headers
+// Windows Header Files
+#include <windows.h>
+
+#include <cstring>
+#include <iostream>
+#include <vector>
+#include <sstream>
+#include <string>
+#include <stdio.h>
+
+#include "bpf2c.h"
+
+#define metadata_table ___METADATA_TABLE___##_metadata_table
+extern metadata_table_t metadata_table;
+
+void
+division_by_zero(uint32_t address)
+{
+    fprintf(stderr, "Divide by zero at address %d\n", address);
+}
+
+metadata_table_t*
+get_metadata_table()
+{
+    return &metadata_table;
+}
+
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
```

Comment:
Why is this here in the middle of the cpp file?  Delete?

Reply:
This was a header that I cut and pasted into the template to make things simpler. Fixed now.